inc/saihostintf.h

```diff
@@ -148,16 +148,26 @@ typedef sai_status_t(*sai_get_hostif_trap_group_attribute_fn)(
         _Inout_ sai_attribute_t *attr_list);
 
 /**
+* @brief Host interface user defined trap type table range
+*/
+#define SAI_HOSTIF_TRAP_TYPE_TABLE_RANGE 0x1
```

Comment:
@lguohan @kcudnik @itaibaz - 
 /**
     * @brief Host interface trap type []
     *
     * @type sai_hostif_trap_type_t
     * @flags MANDATORY_ON_CREATE | CREATE_ONLY | KEY
     */
    SAI_HOSTIF_TRAP_ATTR_TRAP_TYPE = SAI_HOSTIF_TRAP_ATTR_START,

What was meaning of KEY here,  will SAI allow creating only object with this TYPE or it just indication to application like this is input for creating object.